java/swagger-petstore/src/main/java/io/swagger/petstore/exception/ApiException.java

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
either have all java files with copy header (set to 2018, not 2016) or none

Reply:
Done!